pixeltable/catalog/table.py

```diff
@@ -60,7 +62,7 @@ def __hash__(self) -> int:
 
     def _check_is_dropped(self) -> None:
         if self._is_dropped:
-            raise excs.Error(f'{self.display_name()} {self.name} has been dropped')
+            raise excs.Error(f'{self._display_name()} {self._name} has been dropped')
```

Comment:
I was expecting an implementation of get_metadata() for this class that adds the table schema to the SchemaObject get_metadata() output.